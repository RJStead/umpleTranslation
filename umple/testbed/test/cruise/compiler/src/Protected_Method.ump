generate Java "../src-gen-umple";

class ProtectedMethod
{
    protected double foo() { 
        String s = "Hello World!";
        return s;
    }
}
