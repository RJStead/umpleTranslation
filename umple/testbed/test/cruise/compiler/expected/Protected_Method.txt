Protected_Method.ump:7: error: incompatible types: String cannot be converted to double
        return s;
               ^
1 error